src/CabanaPD_Solver.hpp

```diff
@@ -103,13 +104,16 @@ class SolverElastic
     using force_model_type = ForceModel;
     using force_type = Force<exec_space, force_model_type>;
     using comm_type = Comm<particle_type, typename force_model_type::base_model,
-                           typename force_model_type::thermal_type>;
+                           typename particle_type::thermal_type>;
     using neighbor_type =
         Cabana::VerletList<memory_space, Cabana::FullNeighborTag,
                            Cabana::VerletLayout2D, Cabana::TeamOpTag>;
     using neigh_iter_tag = Cabana::SerialOpTag;
     using input_type = InputType;
 
+    // Optional modules.
+    using heat_transfer_type = HeatTransfer<exec_space, force_model_type>;
```

Comment:
Where in this line it says is "optional"?

Reply:
This will only be constructed and used for problems with heat transfer (below). But that's not clear here, so I added the comment